Anlab.Mvc/Extensions/StringExtensions.cs

```diff
@@ -53,5 +54,22 @@ public static bool IsEmailValid(this string value)
             }
             return Regex.IsMatch(value.ToLower(), emailRegex);
         }
+
+        public static string FormatPhone(this string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                return value;
+            }
+            var phoneNumberUtil = PhoneNumbers.PhoneNumberUtil.GetInstance();
+            var phoneNumber = phoneNumberUtil.ParseAndKeepRawInput(value, "US");
+            var phone = phoneNumberUtil.Format(phoneNumber, PhoneNumberFormat.NATIONAL);
+            if (phone == "+1 NA")

```

Comment:
@jSylvestre seems to me it'd be better to check if the phone number is valid instead of checking if the formatted string == "+1 NA".  I'm sure the parsed result has a valid or similar prop on it